.gitignore

```diff
@@ -23,7 +23,6 @@ node_modules
 staticfiles
 project/media
 local_settings.py
-*.pdf
```

Comment:
Now that we have factories for FilledPDF and FillablePDF this ensures the files needed for those factories will be included.